--- gst-debugger-0.90.0/src/common/Makefile.am.orig	2015-10-04 14:26:07.000000000 +0200
+++ gst-debugger-0.90.0/src/common/Makefile.am	2016-04-26 20:19:51.643448366 +0200
@@ -29,11 +29,11 @@
 libgst_debugger_common_c_@GST_DEBUGGER_API_VERSION@_la_SOURCES = $(common_sources) gstdebugger.pb-c.c
 libgst_debugger_common_c_@GST_DEBUGGER_API_VERSION@_lainclude_HEADERS = $(common_headers) gstdebugger.pb-c.h
 libgst_debugger_common_c_@GST_DEBUGGER_API_VERSION@_laincludedir = $(common_include_dir)
-libgst_debugger_common_c_@GST_DEBUGGER_API_VERSION@_la_LDFLAGS = $(PROTOBUF_C_LIBS) $(GSTREAMER_LIBS)
+libgst_debugger_common_c_@GST_DEBUGGER_API_VERSION@_la_LDFLAGS = $(PROTOBUF_C_LIBS) $(GSTREAMER_LIBS) $(GIO_LIBS)
 libgst_debugger_common_c_@GST_DEBUGGER_API_VERSION@_la_CFLAGS = -I.. $(PROTOBUF_C_CFLAGS) $(GSTREAMER_CFLAGS)
 
 libgst_debugger_common_cpp_@GST_DEBUGGER_API_VERSION@_la_SOURCES = $(common_sources) gstdebugger.pb.cc
 libgst_debugger_common_cpp_@GST_DEBUGGER_API_VERSION@_lainclude_HEADERS = $(common_headers) gstdebugger.pb.h
 libgst_debugger_common_cpp_@GST_DEBUGGER_API_VERSION@_laincludedir = $(common_include_dir)
-libgst_debugger_common_cpp_@GST_DEBUGGER_API_VERSION@_la_LDFLAGS  = $(PROTOBUF_LIBS) $(GSTREAMER_LIBS)
+libgst_debugger_common_cpp_@GST_DEBUGGER_API_VERSION@_la_LDFLAGS  = $(PROTOBUF_LIBS) $(GSTREAMER_LIBS) $(GIO_LIBS)
 libgst_debugger_common_cpp_@GST_DEBUGGER_API_VERSION@_la_CFLAGS = -I.. $(PROTOBUF_CFLAGS) $(GSTREAMER_CFLAGS)
